security/maltrail/src/opnsense/service/templates/OPNsense/Maltrail/maltrail.conf

```diff
@@ -0,0 +1,56 @@
+{% from 'OPNsense/Macros/interface.macro' import physical_interface %}
+
+{% if helpers.exists('OPNsense.maltrail.server.enabled') and OPNsense.maltrail.server.enabled == '1' %}
+
+# [Server]
+HTTP_ADDRESS {{ OPNsense.maltrail.server.listenaddress }}
+HTTP_PORT {{ OPNsense.maltrail.server.listenport }}
+USE_SSL false
+
+{%   if helpers.exists('OPNsense.maltrail.server.loglistenaddress') and OPNsense.maltrail.server.loglistenaddress != '' %}
+UDP_ADDRESS {{ OPNsense.maltrail.server.loglistenaddress }}
+{%   endif %}
+{%   if helpers.exists('OPNsense.maltrail.server.loglistenport') and OPNsense.maltrail.server.loglistenport != '' %}
+UDP_PORT {{ OPNsense.maltrail.server.loglistenport }}
+{%   endif %}
+{% endif %}
+
+{% if helpers.exists('OPNsense.maltrail.sensor.enabled') and OPNsense.maltrail.sensor.enabled == '1' %}
+{%   if helpers.exists('OPNsense.maltrail.sensor.remoteserver') and OPNsense.maltrail.sensor.remoteserver != '' %}
+LOG_SERVER {{ OPNsense.maltrail.sensor.remoteserver }}:{{ OPNsense.maltrail.sensor.remoteport }}
+DISABLE_LOCAL_LOG_STORAGE true
+{%   else %}
+DISABLE_LOCAL_LOG_STORAGE false
+{%   endif %}
+{% endif %}
+
+SENSOR_NAME $HOSTNAME
+CUSTOM_TRAILS_DIR /usr/local/maltrail/trails/custom/
+PROCESS_COUNT $CPU_CORES
+DISABLE_CPU_AFFINITY false
+USE_FEED_UPDATES true
+DISABLED_FEEDS turris, ciarmy, policeman, myip
+UPDATE_PERIOD {{ OPNsense.maltrail.general.updateperion }}
+USE_SERVER_UPDATE_TRAILS false
+{% if helpers.exists('OPNsense.maltrail.general.heuristics') and OPNsense.maltrail.general.heuristics == '1' %}
+USE_HEURISTICS true
+{% else %}
+USE_HEURISTICS false
+{% endif %}
+CHECK_MISSING_HOST false
+CHECK_HOST_DOMAINS false
+SHOW_DEBUG false
+LOG_DIR /var/log/maltrail
+{% if helpers.exists('OPNsense.maltrail.general.monitorinterface') and OPNsense.maltrail.general.monitorinterface != '' %}
+{%   set interfaces = [] %}
+{%   for interface in OPNsense.maltrail.general.monitorinterface.split(',') %}
+{%     do interfaces.append(physical_interface(interface)) %}
+{%   endfor %}
+MONITOR_INTERFACE {{ interfaces|join(',') }}
+{% else %}
+MONITOR_INTERFACE any
+{% endif %}
+CAPTURE_BUFFER 10%
+CAPTURE_FILTER udp or icmp or (tcp and (tcp[tcpflags] == tcp-syn or port 80 or port 1080 or port 3128 or port 8000 or port 8080 or port 8118))
```

Comment:
does this make sense?


Sone defaults for some frameworks: 

* sinatrarb - 4567
* rails - 3000
* express.js - 9000 / 9001

Reply:
These are defaults by vendor, as he's also the author of sqlmap I think this should be quite ok.
Perhaps make it user-selectable in one of the next versions if there's some user feedback